crowbar_framework/app/views/repositories/index.html.haml

```diff
@@ -0,0 +1,80 @@
+.col-xs-12
+  .row
+    %h1.page-header
+      = t(".title")
+    .alert.alert-info
+      = t(".refresh_hint")
+
+  - repository_platforms.each_with_index do |platform, index|
+    .row
+      = form_for :repository, :url => repo_sync_path, :html => { :role => "form" } do |f|
+        .panel.panel-default
+          .panel-heading
+            %h2
+              = pretty_target_platform(platform)
+
+          %table.table.table-hover.table-middle.table-bordered
+            %thead
+              %tr
+                %th.col-xs-5
+                  = t(".name")
+                %th.col-xs-2
+                  = t(".required")
+                %th.col-xs-2
+                  = t(".status")
+                %th.col-xs-2
+                  = t(".md5_check")
+                %th.col-xs-1
+                  %input{ :type => "checkbox", :value => "all", "data-checkall" => "input.platform#{index}" }
+                    = t(".active")
+                  .pull-right
+                    %i.glyphicon.glyphicon-question-sign{ :data => { :toggle => "tooltip", :placement => "top" }, :title => t(".active_repo_tooltip") }
```

Comment:
Bah, it seems [there is no good way to avoid long lines like this](https://github.com/haml/haml/issues/677) :-(